metadata.rb

```diff
@@ -16,7 +15,6 @@
 depends          'osl-haproxy'
 depends          'osl-docker'
 depends          'certificate'
-depends          'git'
```

Comment:
This should be changed to ``osl-git`` along with updating the ``include_recipe`` in the ``master`` recipe.